lua/swampshop/cl_draw.lua

```diff
@@ -360,10 +379,11 @@ function SS_AttachAccessory(item, ent, recycle_mdl)
     else
         mdl = ClientsideModel(item:GetModel(), RENDERGROUP_OPAQUE)
     end
-
+    mdl:SetNoDraw(true)
```

Comment:
I'm pretty sure this function is making accessories (on other players) that control their own rendering...?

Reply:
there's seemingly some issue where  props spawned by the menu are not disposed of correctly. i think i need to look into that.